integration/simulation/simulation.go

```diff
@@ -66,14 +66,14 @@ func (s *Simulation) Start() {
 	s.prefundL1Accounts()      // Prefund every L1 wallet
 
 	timer := time.Now()
-	log.Info("Starting injection")
+	fmt.Printf("Starting injection\n")
```

Comment:
It made more sense to print these so they can be seen when running the tests, than to stick them in the logs, IMO. It's quite nice seeing these appear, to show you that the full network sim is making progress.

Reply:
Do both then imo, because it's nice when you have to investigate something in the log file to see when the injection started.